rdflib/graph.py

```diff
@@ -2034,7 +2037,11 @@ def quads(self, quad):
             else:
                 yield s, p, o, c.identifier
 
-    def __iter__(self) -> Generator[DatasetQuad, None, None]:
+    # FIXME: Currently graph identifiers/context can be any Node, even
+    # though this goes outside of the RDF 1.1. specificiation. This should be
+    # fixed, but the fix should not only fix the return value here but also
+    # narrow the type annotations of Graph.__new__.
+    def __iter__(self) -> Generator[Tuple[Node, URIRef, Node, Optional[Node]], None, None]:
```

Comment:
Actually I think I forgot to clean this up in https://github.com/ajnelson-nist/rdflib/pull/13 - this can be narrowed now:

```suggestion
    def __iter__(self) -> Generator[Tuple[Node, URIRef, Node, Optional[IdentifiedNode]], None, None]:
```